src/main/java/nextstep/ladder/factory/LadderFactory.java

```diff
@@ -0,0 +1,12 @@
+package nextstep.ladder.factory;
+
+import nextstep.ladder.domain.LadderImpl;
+import nextstep.ladder.generator.NonConsecutiveFlagGenerator;
+
+public class LadderFactory {
+
+    public static LadderImpl ladder(int height, int width, NonConsecutiveFlagGenerator generator) {
```

Comment:
```suggestion
    public static Ladder ladder(int height, int width, NonConsecutiveFlagGenerator generator) {
```
 인터페이스를 추가한 만큼 구현체가 아닌 인터페이스로 구현해 보면 어떨까?